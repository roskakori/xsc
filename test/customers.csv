id,surname,firstname,dateOfBirth
1,Doe,John,1957-03-08
2,Miller,Jane,1946-10-04
3,Webster,Mike,1974-12-23
