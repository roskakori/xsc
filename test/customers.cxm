<?xml version="1.0" encoding="utf-8"?>
<!-- A simple list of all customers. -->
<customers>
  <person/>
  <?cxm for customers?>
  <person>
    <surname>${customers.surname}</surname>
    <firstname>${customers.surname}</firstname>
    <dateOfBirth>${customers.dateOfBirth}</dateOfBirth>
  </person>
  <?cxm end for?>
</customers>
