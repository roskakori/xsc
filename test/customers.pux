<?xml version="1.0" encoding="utf-8"?>
<!-- A simple list of all customers. -->
<customers>
  <person/>
  <?pux for customers?>
  <person>
    <surname>${customers.surname}</surname>
    <firstname>${customers.surname}</firstname>
    <dateOfBirth>${customers.dateOfBirth}</dateOfBirth>
  </person>
  <?pux end for?>
</customers>
