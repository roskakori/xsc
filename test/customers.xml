<?xml version="1.0" encoding="utf-8"?><!-- A simple list of all customers. --><customers>
  
  <person>
    