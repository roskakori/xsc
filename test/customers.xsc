<?xml version="1.0" encoding="utf-8"?>
<!-- A simple list of all customers. -->
<customers>
  <person/>
  <?xsc for customers?>
  <person>
    <surname>${customers.surname}</surname>
    <firstname>${customers.surname}</firstname>
    <dateOfBirth>${customers.dateOfBirth}</dateOfBirth>
  </person>
  <?xsc end for?>
</customers>
