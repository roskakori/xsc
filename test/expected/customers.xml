<?xml version="1.0" encoding="utf-8"?><!-- A simple list of all customers. --><customers>
  <person/>
  
  <person>
    <surname>Doe</surname>
    <firstname>Doe</firstname>
    <dateOfBirth>1957-03-08</dateOfBirth>
  </person>
  
  <person>
    <surname>Miller</surname>
    <firstname>Miller</firstname>
    <dateOfBirth>1946-10-04</dateOfBirth>
  </person>
  
  <person>
    <surname>Webster</surname>
    <firstname>Webster</firstname>
    <dateOfBirth>1974-12-23</dateOfBirth>
  </person>
  
</customers>